bindings/pyroot/pythonizations/python/ROOT/_pythonization/_function_utils.py

```diff
@@ -0,0 +1,224 @@
+import re
+from .._numbadeclare import _NumbaDeclareDecorator
+
+class FunctionSignature():
+    """
+    Class to determine a Function's Signature.
+    """
+    function_cache = {} # Variable to store previous functions so as to not rejit.
```

Comment:
The comment here should go in the docstring of the class, in the section where the data members are described.